File: java-ConsumoAPI/src/main/java/com/mycompany/consumoapi/models/BreedCat.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.consumoapi.models;

import com.mycompany.consumoapi.utils.StringManager;

/**
 *
 * @author kikem
 */
public class BreedCat {
    int id;
    String idName;
    String name;
    String name_es;
    //criado para
    String cfa_url;
    //grupo de raza
    String vetstreet_url;
     //esperanza de vida
    String vcahospitals_url;

    String temperament;
    
    String temperament_es;

    String origin;

    String country_codes;

    String country_code;

    String description;
    
    String description_es;

    String life_span;

    int indoor;

    int lap;

    String alt_names;

    int adaptability;

    int affection_level;

    int child_friendly;

    int dog_friendly;
    
    int energy_level;

    int grooming;

    int health_issues;

    int intelligence;

    int shedding_level;

    int social_needs;

    int stranger_friendly;

    int vocalisation;
    
    int experimental;

    int hairless;

    int natural;

    int rare;

    int rex;

    int suppressed_tail;

    int short_legs;

    String wikipedia_url;

    int hypoallergeni;

    String weight;

    String date;

    String reference_image_id;
    
    String path_image;
    
    int creator_id;

    public BreedCat(int id) {
        this.id = id;
    }

    public BreedCat(int id, String idName, String name, String name_es, String cfa_url, String vetstreet_url, String vcahospitals_url, String temperament, String temperament_es, String origin, String country_codes, String country_code, String description, String description_es, String life_span, int indoor, int lap, String alt_names, int adaptability, int affection_level, int child_friendly, int dog_friendly, int energy_level, int grooming, int health_issues, int intelligence, int shedding_level, int social_needs, int stranger_friendly, int vocalisation, int experimental, int hairless, int natural, int rare, int rex, int suppressed_tail, int short_legs, String wikipedia_url, int hypoallergeni, String weight, String date, String reference_image_id, String path_image, int creator_id) {
        this.id = id;
        this.idName = idName;
        this.name = name;
        this.name_es = name_es;
        this.cfa_url = cfa_url;
        this.vetstreet_url = vetstreet_url;
        this.vcahospitals_url = vcahospitals_url;
        this.temperament = temperament;
        this.temperament_es = temperament_es;
        this.origin = origin;
        this.country_codes = country_codes;
        this.country_code = country_code;
        this.description = description;
        this.description_es = description_es;
        this.life_span = life_span;
        this.indoor = indoor;
        this.lap = lap;
        this.alt_names = alt_names;
        this.adaptability = adaptability;
        this.affection_level = affection_level;
        this.child_friendly = child_friendly;
        this.dog_friendly = dog_friendly;
        this.energy_level = energy_level;
        this.grooming = grooming;
        this.health_issues = health_issues;
        this.intelligence = intelligence;
        this.shedding_level = shedding_level;
        this.social_needs = social_needs;
        this.stranger_friendly = stranger_friendly;
        this.vocalisation = vocalisation;
        this.experimental = experimental;
        this.hairless = hairless;
        this.natural = natural;
        this.rare = rare;
        this.rex = rex;
        this.suppressed_tail = suppressed_tail;
        this.short_legs = short_legs;
        this.wikipedia_url = wikipedia_url;
        this.hypoallergeni = hypoallergeni;
        this.weight = weight;
        this.date = date;
        this.reference_image_id = reference_image_id;
        this.path_image = path_image;
        this.creator_id = creator_id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getIdName() {
        return idName;
    }

    public void setIdName(String idName) {
        this.idName = idName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName_es() {
        return name_es;
    }

    public void setName_es(String name_es) {
        this.name_es = name_es;
    }

    public String getCfa_url() {
        return cfa_url;
    }

    public void setCfa_url(String cfa_url) {
        this.cfa_url = cfa_url;
    }

    public String getVetstreet_url() {
        return vetstreet_url;
    }

    public void setVetstreet_url(String vetstreet_url) {
        this.vetstreet_url = vetstreet_url;
    }

    public String getVcahospitals_url() {
        return vcahospitals_url;
    }

    public void setVcahospitals_url(String vcahospitals_url) {
        this.vcahospitals_url = vcahospitals_url;
    }

    public String getTemperament() {
        return temperament;
    }

    public void setTemperament_es(String temperament_es) {
        this.temperament_es = temperament_es;
    }
        public String getTemperament_es() {
        return temperament_es;
    }

    public void setTemperament(String temperament) {
        this.temperament = temperament;
    }

    

    public String getOrigin() {
        return origin;
    }

    public void setOrigin(String origin) {
        this.origin = origin;
    }

    public String getCountry_codes() {
        return country_codes;
    }

    public void setCountry_codes(String country_codes) {
        this.country_codes = country_codes;
    }

    public String getCountry_code() {
        return country_code;
    }

    public void setCountry_code(String country_code) {
        this.country_code = country_code;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDescription_es() {
        return description_es;
    }

    public void setDescription_es(String description_es) {
        this.description_es = description_es;
    }

    public String getLife_span() {
        return life_span;
    }

    public void setLife_span(String life_span) {
        this.life_span = life_span;
    }

    public int getIndoor() {
        return indoor;
    }

    public void setIndoor(int indoor) {
        this.indoor = indoor;
    }

    public int getLap() {
        return lap;
    }

    public void setLap(int lap) {
        this.lap = lap;
    }

    public String getAlt_names() {
        return alt_names;
    }

    public void setAlt_names(String alt_names) {
        this.alt_names = alt_names;
    }

    public int getAdaptability() {
        return adaptability;
    }

    public void setAdaptability(int adaptability) {
        this.adaptability = adaptability;
    }

    public int getAffection_level() {
        return affection_level;
    }

    public void setAffection_level(int affection_level) {
        this.affection_level = affection_level;
    }

    public int getChild_friendly() {
        return child_friendly;
    }

    public void setChild_friendly(int child_friendly) {
        this.child_friendly = child_friendly;
    }

    public int getDog_friendly() {
        return dog_friendly;
    }

    public void setDog_friendly(int dog_friendly) {
        this.dog_friendly = dog_friendly;
    }

    public int getEnergy_level() {
        return energy_level;
    }

    public void setEnergy_level(int energy_level) {
        this.energy_level = energy_level;
    }

    public int getGrooming() {
        return grooming;
    }

    public void setGrooming(int grooming) {
        this.grooming = grooming;
    }

    public int getHealth_issues() {
        return health_issues;
    }

    public void setHealth_issues(int health_issues) {
        this.health_issues = health_issues;
    }

    public int getIntelligence() {
        return intelligence;
    }

    public void setIntelligence(int intelligence) {
        this.intelligence = intelligence;
    }

    public int getShedding_level() {
        return shedding_level;
    }

    public void setShedding_level(int shedding_level) {
        this.shedding_level = shedding_level;
    }

    public int getSocial_needs() {
        return social_needs;
    }

    public void setSocial_needs(int social_needs) {
        this.social_needs = social_needs;
    }

    public int getStranger_friendly() {
        return stranger_friendly;
    }

    public void setStranger_friendly(int stranger_friendly) {
        this.stranger_friendly = stranger_friendly;
    }

    public int getVocalisation() {
        return vocalisation;
    }

    public void setVocalisation(int vocalisation) {
        this.vocalisation = vocalisation;
    }

    public int getExperimental() {
        return experimental;
    }

    public void setExperimental(int experimental) {
        this.experimental = experimental;
    }

    public int getHairless() {
        return hairless;
    }

    public void setHairless(int hairless) {
        this.hairless = hairless;
    }

    public int getNatural() {
        return natural;
    }

    public void setNatural(int natural) {
        this.natural = natural;
    }

    public int getRare() {
        return rare;
    }

    public void setRare(int rare) {
        this.rare = rare;
    }

    public int getRex() {
        return rex;
    }

    public void setRex(int rex) {
        this.rex = rex;
    }

    public int getSuppressed_tail() {
        return suppressed_tail;
    }

    public void setSuppressed_tail(int suppressed_tail) {
        this.suppressed_tail = suppressed_tail;
    }

    public int getShort_legs() {
        return short_legs;
    }

    public void setShort_legs(int short_legs) {
        this.short_legs = short_legs;
    }

    public String getWikipedia_url() {
        return wikipedia_url;
    }

    public void setWikipedia_url(String wikipedia_url) {
        this.wikipedia_url = wikipedia_url;
    }

    public int getHypoallergeni() {
        return hypoallergeni;
    }

    public void setHypoallergeni(int hypoallergeni) {
        this.hypoallergeni = hypoallergeni;
    }

    public String getWeight() {
        return weight;
    }

    public void setWeight(String weight) {
        this.weight = weight;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getReference_image_id() {
        return reference_image_id;
    }

    public void setReference_image_id(String reference_image_id) {
        this.reference_image_id = reference_image_id;
    }

    public String getPath_image() {
        return path_image;
    }

    public void setPath_image(String path_image) {
        this.path_image = path_image;
    }

    public int getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(int creator_id) {
        this.creator_id = creator_id;
    }

    @Override
    public String toString() {
        return "BreedCat{" + "id=" + id + ", idName=" + idName + ", name=" + name + ", name_es=" + name_es + ", cfa_url=" + cfa_url + ", vetstreet_url=" + vetstreet_url + ", vcahospitals_url=" + vcahospitals_url + ", temperament=" + temperament + ", origin=" + origin + ", country_codes=" + country_codes + ", country_code=" + country_code + ", description=" + description + ", description_es=" + description_es + ", life_span=" + life_span + ", indoor=" + indoor + ", lap=" + lap + ", alt_names=" + alt_names + ", adaptability=" + adaptability + ", affection_level=" + affection_level + ", child_friendly=" + child_friendly + ", dog_friendly=" + dog_friendly + ", energy_level=" + energy_level + ", grooming=" + grooming + ", health_issues=" + health_issues + ", intelligence=" + intelligence + ", shedding_level=" + shedding_level + ", social_needs=" + social_needs + ", stranger_friendly=" + stranger_friendly + ", vocalisation=" + vocalisation + ", experimental=" + experimental + ", hairless=" + hairless + ", natural=" + natural + ", rare=" + rare + ", rex=" + rex + ", suppressed_tail=" + suppressed_tail + ", short_legs=" + short_legs + ", wikipedia_url=" + wikipedia_url + ", hypoallergeni=" + hypoallergeni + ", weight=" + weight + ", date=" + date + ", reference_image_id=" + reference_image_id + ", path_image=" + path_image + ", creator_id=" + creator_id + '}';
    }

   
   

    
}

File: php-breeds/app/models/BreedCat.php
<?php

Class BreedCat {
	private $id;
	private $id_name;
	private $name;
	private $name_es;
	private $cfa_url;
	private $vetstreet_url;
	private $vcahospitals_url;
	private $temperament;
	private $temperament_es;
	private $origin;
	private $origin_es;
	private $country_codes;
	private $country_code;
	private $description;
	private $description_es;
	private $morphology;
	private $morphology_es;
	private $life_span;
	//interior
	private $indoor;
	//regazo
	private $lap;
	//nombres alternativos
	private $alt_names;
  	private $adaptability;
  	private $affection_level;
  	private $child_friendly;
  	private $cat_friendly;
  	private $dog_friendly;
  	private $energy_level;
	//aseo
	private $grooming;
	//problemas de salud
	private $health_issues;
	private $intelligence;
	//nivel de muda
    private $shedding_level;
	//necesidades sociales
	private $social_needs;
	private $stranger_friendly;
	//vocalización
  	private $vocalisation;
  	private $experimental;
	//sin pelo
	private $hairless;
  	private $natural;
  	private $rare;
  	private $rex;
	//cola suprimida
	private $suppressed_tail;
	//piernas cortas
  	private $short_legs;
  	private $wikipedia_url;
	//hipoalergénica
	private $hypoallergenic;
	//peso
	private $weight;
	//Las razas pueden tener varias imágenes
  	private $date;
  	private $reference_image_id;
	private $path_image;
  	private $creator_id;




	
	/**
	 * CONSTRUCTOR
	 */
	function __construct($id){
		$this->id=$id;
	}
	

	/**
	 * SETTERS
	 */
	function set_id_name($id_name){
		$this->id_name=$id_name;
	}
	function set_name($name){
		$this->name=$name;
	}
	function set_name_es($name_es){
		$this->name_es=$name_es;
	}
	function set_cfa_url($cfa_url){
		$this->cfa_url=$cfa_url;
	}
	function set_vetstreet_url($vetstreet_url){
		$this->vetstreet_url=$vetstreet_url;
	}
	function set_vcahospitals_url($vcahospitals_url){
		$this->vcahospitals_url=$vcahospitals_url;
	}
	function set_temperament($temperament){
		$this->temperament=$temperament;
	}
	function set_temperament_es($temperament_es){
		$this->temperament_es=$temperament_es;
	}
	function set_origin($origin){
		$this->origin=$origin;
	}
	function set_origin_es($origin_es){
		$this->origin_es=$origin_es;
	}
	function set_country_codes($country_codes){
		$this->country_codes=$country_codes;
	}
	function set_country_code($country_code){
		$this->country_code=$country_code;
	}
	function set_description($description){
		$this->description=$description;
	}
	function set_description_es($description_es){
		$this->description_es=$description_es;
	}
	function set_morphology($morphology){
		$this->morphology=$morphology;
	}
	function set_morphology_es($morphology_es){
		$this->morphology_es=$morphology_es;
	}
	function set_life_span($life_span){
		$this->life_span=$life_span;
	}
	function set_indoor($indoor){
		$this->indoor=$indoor;
	}
	function set_lap($lap){
		$this->lap=$lap;
	}
	function set_alt_names($alt_names){
		$this->alt_names=$alt_names;
	}
	function set_adaptability($adaptability){
		$this->adaptability=$adaptability;
	}
	function set_affection_level($affection_level){
		$this->affection_level=$affection_level;
	}
	function set_child_friendly($child_friendly){
		$this->child_friendly=$child_friendly;
	}
	function set_cat_friendly($cat_friendly){
		$this->cat_friendly=$cat_friendly;
	}
	function set_dog_friendly($dog_friendly){
		$this->dog_friendly=$dog_friendly;
	}
	function set_energy_level($energy_level){
		$this->energy_level=$energy_level;
	}
	function set_grooming($grooming){
		$this->grooming=$grooming;
	}
	function set_health_issues($health_issues){
		$this->health_issues=$health_issues;
	}
	function set_intelligence($intelligence){
		$this->intelligence=$intelligence;
	}
	function set_shedding_level($shedding_level){
		$this->shedding_level=$shedding_level;
	}
	function set_social_needs($social_needs){
		$this->social_needs=$social_needs;
	}
	function set_stranger_friendly($stranger_friendly){
		$this->stranger_friendly=$stranger_friendly;
	}
	function set_vocalisation($vocalisation){
		$this->vocalisation=$vocalisation;
	}
	function set_experimental($experimental){
		$this->experimental=$experimental;
	}
	function set_hairless($hairless){
		$this->hairless=$hairless;
	}
	function set_natural($natural){
		$this->natural=$natural;
	}
	function set_rare($rare){
		$this->rare=$rare;
	}
	function set_rex($rex){
		$this->rex=$rex;
	}
	function set_suppressed_tail($suppressed_tail){
		$this->suppressed_tail=$suppressed_tail;
	}
	function set_short_legs($short_legs){
		$this->short_legs=$short_legs;
	}
	function set_wikipedia_url($wikipedia_url){
		$this->wikipedia_url=$wikipedia_url;
	}
	function set_hypoallergenic($hypoallergenic){
		$this->hypoallergenic=$hypoallergenic;
	}
	function set_weight($weight){
		$this->weight=$weight;
	}
	function set_date($date){
		$this->date=$date;
	}
	function set_reference_image_id($reference_image_id){
		$this->reference_image_id=$reference_image_id;
	}
	function set_path_image($path_image){
		$this->path_image=$path_image;
	}
	function set_creator_id($creator_id){
		$this->creator_id=$creator_id;
	}



	








	/**
	 * GETTERS
	 */
	function get_id(){
		return $this->id;
	}
	function get_id_name(){
		return $this->id_name;
	}
	function get_name(){
		return $this->name;
	}
	function get_name_es(){
		return $this->name_es;
	}
	function get_cfa_url(){
		return $this->cfa_url;
	}
	function get_vetstreet_url(){
		return $this->vetstreet_url;
	}
	function get_vcahospitals_url(){
		return $this->vcahospitals_url;
	}
	function get_temperament(){
		return $this->temperament;
	}
	function get_temperament_es(){
		return $this->temperament_es;
	}
	function get_origin(){
		return $this->origin;
	}
	function get_origin_es(){
		return $this->origin_es;
	}
	function get_country_codes(){
		return $this->country_codes;
	}
	function get_country_code(){
		return $this->country_code;
	}
	function get_description(){
		return $this->description;
	}
	function get_description_es(){
		return $this->description_es;
	}
	function get_morphology(){
		return $this->morphology;
	}
	function get_morphology_es(){
		return $this->morphology_es;
	}
	function get_life_span(){
		return $this->life_span;
	}
	function get_indoor(){
		return $this->indoor;
	}
	function get_lap(){
		return $this->lap;
	}
	function get_alt_names(){
		return $this->alt_names;
	}
	function get_adaptability(){
		return $this->adaptability;
	}
	function get_affection_level(){
		return $this->affection_level;
	}
	function get_child_friendly(){
		return $this->child_friendly;
	}
	function get_cat_friendly(){
		return $this->cat_friendly;
	}
	function get_dog_friendly(){
		return $this->dog_friendly;
	}
	function get_energy_level(){
		return $this->energy_level;
	}
	function get_grooming(){
		return $this->grooming;
	}
	function get_health_issues(){
		return $this->health_issues;
	}
	function get_intelligence(){
		return $this->intelligence;
	}
	function get_shedding_level(){
		return $this->shedding_level;
	}
	function get_social_needs(){
		return $this->social_needs;
	}
	function get_stranger_friendly(){
		return $this->stranger_friendly;
	}
	function get_vocalisation(){
		return $this->vocalisation;
	}
	function get_experimental(){
		return $this->experimental;
	}
	function get_hairless(){
		return $this->hairless;
	}
	function get_natural(){
		return $this->natural;
	}
	function get_rare(){
		return $this->rare;
	}
	function get_rex(){
		return $this->rex;
	}
	function get_suppressed_tail(){
		return $this->suppressed_tail;
	}
	function get_short_legs(){
		return $this->short_legs;
	}
	function get_wikipedia_url(){
		return $this->wikipedia_url;
	}
	function get_hypoallergenic(){
		return $this->hypoallergenic;
	}
	function get_weight(){
		return $this->weight;
	}
	function get_date(){
		return $this->date;
	}
	function get_reference_image_id(){
		return $this->reference_image_id;
	}
	function get_path_image(){
		return $this->path_image;
	}
	function get_creator_id(){
		return $this->creator_id;
	}


	
	
	
	/**
	 * TODTRING
	 */
	function to_string(){
		$path=$this->path_image;
		$order   = array("\\", "\\\\", "\\\\\\");
		$path=str_replace($order,"/",$path);
		return"<br><img src=".PATHSERVER.$path." width=40px>Cat id: ".$this->id.", name: ".$this->name.", path: ".$this->path_image.", creator: ".$this->creator_id;	
	}

	
	
}

?>